src/components/composite/Product-cards/Item-Hooks/add-wish.js

```diff
@@ -1,7 +1,29 @@
-const useAddWishItem = (product, dispatchWish) =>
-    dispatchWish({
-        type: 'ADD_TO_WISHLIST',
-        payload: product,
-    })
+// const useAddWishItem = (product, dispatchWish) =>
+import axios from 'axios'
+
+const getConfig = (token) => ({
```

Comment:
You can extract out get config to some helper function and then use it every where Ratner than repeating it

Reply:
Yup will do that in the next pr. Thanks for the suggestion